container/build.py

```diff
@@ -26,45 +42,52 @@
 $ python build.py -d {container_type} -l
 will produce docker locally as {container_type}:latest
 
-usage:
-  build.py -d {rbe-debian8,rbe-debian9,rbe-ubuntu16_04,ubuntu16_04-bazel-docker,ubuntu16_04-bazel}
-      [-p PROJECT] [-c CONTAINER] [-t TAG] [-a] [-b BUCKET] [-h]
-      [-l]
-
 required arguments:
-  -d TYPE, --type TYPE  Type of the container: see TYPE_TARGET_MAP
+  -d, --type            Type of the container: see SUPPORTED_TYPES
+required arguments (for cloud build):
   -p PROJECT, --project PROJECT
                         GCP project ID
   -c CONTAINER, --container CONTAINER
                         Docker container name
   -t TAG, --tag TAG     Docker tag for the image
+  -v BAZEL_VERSION, --bazel_version BAZEL_VERSION
+                        The version of Bazel to build the image with on Google
+                        Cloud Container Builder, e.g. 0.15.1
 
 optional arguments:
+  -h, --help            print this help text and exit
+
+optional arguments (for cloud build):
   -a, --async           Asynchronous execute Cloud Container Builder
   -b BUCKET, --bucket BUCKET
                         GCS bucket to store the tarball of debian packages
-  -h, --help            print this help text and exit
+  -m MAP, --map MAP     path to override target map file (can be absolute or
+                        relative). This allows the building of bazel targets
```

Comment:
r/building

Reply:
path (can be absolute or relative) to file containing 3 maps to
override the default ones defined below (TYPE_PACKAGE_MAP, TYPE_TARGET_MAP, and TYPE_TARBALL_MAP)

Reply:
Changed it to use this help text